client/my-sites/hosting/github/github-authorize-card/index.tsx

```diff
@@ -30,7 +30,7 @@ export const GithubAuthorizeCard = () => {
 	return (
 		<Card className="github-hosting-card">
 			<img className="github-hosting-icon" src={ iconGitHub } alt="" />
-			<CardHeading>{ translate( 'Connect GitHub' ) }</CardHeading>
+			<CardHeading id="connect-github">{ translate( 'Connect GitHub' ) }</CardHeading>
```

Comment:
`#connect-github` doesn't seem to work for me...


https://user-images.githubusercontent.com/36432/218180309-5eb09a31-dacf-4869-acd5-ec7f1618040d.mp4

However, Web Server Settings does seem to work:



https://user-images.githubusercontent.com/36432/218180624-a270e34a-cc52-4254-853c-2daf71ecd918.mp4

Any ideas what the difference might be?

Reply:
Not sure, I've just checked it again in Chrome and Safari, and it worked well for all cases.

Reply:
I couldn't reproduce the issue. Maybe there was a console error with more info. I tried to break it using Fast 3G, but it is still working.
It works for me for each link 👌 .